src/rgw/rgw_asio_frontend.cc

```diff
@@ -587,17 +592,17 @@ void AsioFrontend::accept(Listener& l, boost::system::error_code ec)
         auto c = connections.add(conn);
         // wrap the socket in an ssl stream
         ssl::stream<tcp::socket&> stream{s, *ssl_context};
-        boost::beast::flat_buffer buffer;
+        auto buffer = std::make_unique<parse_buffer>();
```

Comment:
@cbodley `std::make_unique` is C++17-only, right? Is that OK for mimic and luminous?

Reply:
its from c++14, which appears to be fine for mimic but not luminous. in luminous, there's a src/common/backport14.h that provides `ceph::make_unique`